tests/manager/api/test_monitor.py

```diff
@@ -193,11 +193,9 @@ def test_reaping(
         call_args_list = (
             services_fixture.analytics_fixture.analytics_mock.collect_event.call_args_list
         )
-        assert len(call_args_list) == 4
+        assert len(call_args_list) == 2
```

Comment:
I don't think the comment on l. 192 makes sense anymore, in that you can no longer establish the order of anything. Might be worth changing that comment to note that we don't expect events from the reaped loans.